tsconfig.json

```diff
@@ -1,13 +1,21 @@
 {
   "compilerOptions": {
     "outDir": "./dist/",
```

Comment:
Could you explain the reason that the main tsconfig output to `dist/` while declaration file to `build/`?